src/content/guides/how-to-register-an-ethereum-account/index.md

```diff
@@ -0,0 +1,49 @@
+---
+title: How to Register an Ethereum account?
+description: A step-by-step guide on Ethereum account creation using a wallet.
+lang: en
+---
+
+# How to: "Register" an Ethereum account
+
+Creating an Ethereum account is very easy and anyone can do it for free with a specific type of app commonly referred to as a wallet. With a wallet you can then hold, send and receive crypto and connect to crypto projects that will enable you to trade NFTs, exchange coins, access play-to-earn games, unlock Defi and much more. 
+
+## Step 1: Browse our curated list of wallets
+
+A wallet is like an online bank account for Ethereum. There are dozens of different wallets to choose from: mobile, desktop or even browser extensions. We have prepared a list of trusted wallets that may be a good place to start. 
+
+<ButtonLink to="/wallets/find-wallet/">
+  Find a wallet
+</ButtonLink>
+
+## Step 2: Choose a wallet that suits your needs
+
+If you are new, you can activate the “New to crypto” filter to view only those wallets that include all necessary features we think would be particularly suitable for beginners. There are also other profile filters to cater to your needs.
+
+## Step 3: Download and install your wallet app
+
+Once you have decided on a specific wallet, visit their website, download and install it. All of them should be free. 
+
+## Step 4: Open the app and generate or import your Ethereum account
+The first time you open your new wallet you might be asked to choose between creating a new account or importing an existing one. Click on the new account creation. 
+
+
+## Step 5: Store your recovery phrase
+
+Some apps will request you to save a recovery phrase. This is extremely important as the person who knows this secret phrase can take control of your account. Never share this with anyone. This phrase should contain 12 randomly generated words (the order of the words matter).
+
+Once you have saved your recovery phrase you should see your wallet dashboard with your balance. Check out our Guide: [How to use a wallet.](https://ethereum.org/en/guides/how-to-use-a-wallet)
+
+
+### FAQ
+
+**Are my wallet and my Ethereum account the same?**
+- No, wallets have the ability to create a new Ethereum account, but the account lives on Ethereum and wallets connect to it. So if you lose your wallet you can still recover your account with the recovery phrase.
+
+**Can I send Bitcoin to an Ethereum address and vice versa?**
+
+- No you cannot, you need two different accounts.
+
+**Is having my own wallet safer than keeping my funds on an exchange?**
+
+- Yes, this is a much safer option because nobody else will have access to your funds. There are unfortunately many examples of failed exchanges that filed for bankrupcy and lost users' savings. Hacks, frozen accounts or blocked withdrawals are some other common risks. Owning a wallet (with a recovery phrase) is the best way to safeguard your assets. 
```

Comment:
1. This doesn't really make the point as to why self-custody is better
2. Do we think flagging bankruptcy/hacks is a good way to educate users here? Seems scary. Maybe it's necessary...

Reply:
i think here it makes sense to talk about it, otherwise whats the difference to the user? what other point could we make here?